$OpenBSD: patch-src_json_c,v 1.1 2020/09/08 17:46:58 tb Exp $

Remove unneeded comparison to fix gcc 4.2.1 warning breaking the sparc64 build
https://github.com/jorisvink/kore/commit/f6af4a27f49d2fef33783066771fc0635288ecc5

Index: src/json.c
--- src/json.c.orig
+++ src/json.c
@@ -780,7 +780,7 @@ json_get_string(struct kore_json *json)
 		if (ch == '"')
 			break;
 
-		if (ch >= 0 && ch <= 0x1f)
+		if (ch <= 0x1f)
 			goto cleanup;
 
 		if (ch == '\\') {
